stdlib/public/core/UnicodeScalar.swift

```diff
@@ -485,3 +485,74 @@ extension Unicode.Scalar {
   }
 }
 
+// Interoperability with FixedWidthIntegers.
+
+extension FixedWidthInteger {
+  /// Construct any FixedWidthInteger with value `v.value`.
+  ///
+  /// - Precondition: `v.value` can be represented as ASCII (0..<128).
+  @inlinable
+  @available(swift 5.1)
+  public init(ascii v: Unicode.Scalar) {
+    _precondition(v.value < 128,
+                  "Code point value does not fit into ASCII")
+    self = Self(v.value)
+  }
+}
+
+extension Array where Element: FixedWidthInteger {
```

Comment:
I think this code should extend `RangeReplaceableCollection` instead.

Reply:
Done, Thanks!